using_osc.mpb

```diff
@@ -1,3 +1,6 @@
 project {
   includes += $(OSC_ROOT)
+
+  libpaths += /usr/lib/x86_64-linux-gnu
```

Comment:
From the oscpack, the PREFIX is set to /usr/local. In my oscpack install, it goes to 

/usr/local/lib/
liboscpack.so  liboscpack.so.1.1.0

and adds the includes to /usr/local/include.

I think we just need to add the libs and not this, right?